design/20230302.gomod.md

```diff
@@ -1,9 +1,58 @@
 # Design: More Modules
 
-NB: This design doc follows from a Hackathon by @SgtCoDFish and @inteon. The intention here is to describe what we did
-and what we discovered, with an eye to seeking consensus and merging upstream.
+NB: This design doc follows from a Hackathon by [@SgtCoDFish](https://github.com/SgtCoDFish) and [@inteon](https://github.com/inteon).
 
-## Problem Statement
+The intention here is to describe what we did and what we discovered, with an eye to seeking consensus and merging upstream.
+
+## In Short
+
+### Assumptions / Axioms
+
+- It's hard or impossible to upgrade our dependencies months after a release
+- We won't change our conservative approach to backports
+- Fewer dependencies in our core go module is a good thing
```

Comment:
```suggestion
- Fewer dependencies in our core go modules are a good thing
```
? (the two changes here are unrelated, the second change is to match `dependencies` and the first to match `modules` on the next line)

Reply:
I think this is slightly different to what I was going for, because I was intentionally only talking about the core module (i.e. `github.com/cert-manager/cert-manager`). I've tried to keep "core module" and that module synonymous.

That said, you're totally right - this proposal also reduces the dependency surface of other modules, too (i.e. the controller no longer has anything to do with helm).

I'll update to be more generic for that reason! Thanks :rocket: